infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -801,6 +802,195 @@ all the operations for the connector:
 
 EOT
     },
+    confluence-cloud = {
+      source_kind : "confluence"
+      availability : "beta"
+      enable_by_default : false
+      worklytics_connector_id : "confluence-cloud-psoxy"
+      target_host : "api.atlassian.com"
+      source_auth_strategy : "oauth2_refresh_token"
+      display_name : "Confluence Cloud"
+      worklytics_connector_name : "Confluence Cloud via Psoxy"
+      secured_variables : [
+        {
+          name : "ACCESS_TOKEN"
+          writable : true
+          sensitive : true
+          value_managed_by_tf : false
+        },
+        {
+          name : "REFRESH_TOKEN"
+          writable : true
+          sensitive : true
+          value_managed_by_tf : false
+        },
+        local.standard_config_values.oauth_refresh_token_lock,
+        {
+          name : "CLIENT_ID"
+          writable : false
+          sensitive : true # not really, but simpler this way; and some may want it treated as sensitive, since would be req'd to brute-force app tokens or something
+          value_managed_by_tf : false
+        },
+        {
+          name : "CLIENT_SECRET"
+          writable : false
+          sensitive : true
+          value_managed_by_tf : false
+        }
+      ],
+      environment_variables : {
+        GRANT_TYPE : "refresh_token"
+        REFRESH_ENDPOINT : "https://auth.atlassian.com/oauth/token"
+        USE_SHARED_TOKEN : "TRUE"
+      }
+      settings_to_provide = {
+      }
+      reserved_concurrent_executions : null
+      enable_side_output : false
+      example_api_calls_user_to_impersonate : null
+      example_api_calls : [
+        "/oauth/token/accessible-resources", # obtain Confluence Cloud ID from here
```

Comment:
OK, but possibly not helpful to list first, just bc it's going to be uninteresting test call that is going to be used by default